src/reducers/postSlice.js

```diff
@@ -69,7 +69,7 @@ export const deletePost = createAsyncThunk(
 export const editPost = createAsyncThunk(
   "posts/editPost",
   async (post, { rejectWithValue }) => {
-    console.log(post)
+
     try {
       const token = localStorage.getItem("token");
```

Comment:
Instead of using the whole statement everywhere, make a function for getting token from localStorage and use that everywhere.